docs/guide/tool-gii.md

```diff
@@ -94,7 +94,7 @@ if (YII_ENV_DEV) {
     $config['modules']['debug'] = 'yii\debug\Module';
 
     $config['bootstrap'][] = 'gii';
-    $config['modules']['gii'] = 'yii\gii\Module'; // <--- here
+    $config['modules']['gii']['class'] = 'yii\gii\Module'; // <--- here
```

Comment:
The original code is fine because you can just specify a class name string instead of a configuration array. 
Your fix is also fine. However, for consistency it's better to do the same for `debug` as well.